packages/atomic/src/components/ipx/atomic-ipx-refine-toggle/atomic-ipx-refine-toggle.pcss

```diff
@@ -0,0 +1,16 @@
+@import '../../../global/global.pcss';
+
+atomic-icon-button {
+  height: 100%;
+  &::part(button) {
+    aspect-ratio: 1 / 1;
+    width: 3.6rem;
+    height: 100%;
+  }
+  &::part(badge) {
+    width: 1.25rem;
+    height: 1.25rem;
+    line-height: 1.25rem;
+    font-size: 0.7rem;
+  }
+}
```

Comment:
Can this CSS be shared with the insight toggle?

Reply:
Not sure I get your question 🤔

Reply:
Ah I don't see any style for `atomic-insight-refine-toggle`, forget about this comment :D